internal/utils/zerologConverter.go

```diff
@@ -0,0 +1,33 @@
+// OS interface wrappers
+
+package utils
+
+import (
+	"log"
+	"strings"
+
+	"github.com/rs/zerolog"
+)
+
+// since the are a couple of places that take different types of loggers outside of our scope, we introduce these simple wrapper to adapt between the worlds.
+
+// ToZeroLogDebug is an io.Writer that can be used for example in log.Logger to write to an existing zerolog writer
+type ToZeroLogDebug struct {
```

Comment:
any change to use `LevelWriter` (that's zerolog) and something like that in the log framework to not loose level information?

Reply:
Since we need this to be an io.Writer to fit into log.Logger, there won't be any level information available.